guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,271 @@
 
-In the *Properties* table of the host details' page, you can view both the global host status and all sub-statuses.
+Build::
+This sub-status is only relevant for hosts provisioned from {Project} or hosts registered through global registration.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Installed | OK | 0
+| Pending installation | OK | 1
+| Token expired | Error | 2
+| Installation error | Error | 3
+|===
 
-Each sub-status can define its own set of possible values that are mapped to the three global status values.
+ifndef::foreman-deb[]
+Compliance::
+Indicates if the host is compliant with OpenSCAP policies.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Compliant | OK | 0
+| Inconclusive | Warning | 1
+| At least one incompliant | Error | 2
+|===
+endif::[]
 
-The *Build* sub-status has two possible values {endash} *pending* and *built* that are both mapped to global *OK* value.
+ifdef::satellite,orcharhino[]
+OVAL scan::
+Indicates if there are any vulnerabilities found on the host
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| No vulnerabilities found | OK | 0
+| Vulnerabilities found | Warning | 1
+| Vulnerabilities with available patch found | Error | 2
+|===
+endif::[]
 
-The *Configuration* status has more possible values that map to the global status as follows:
+Execution::
+Status of the last completed remote excution job.
+ifdef::foreman-el,foreman-deb[]
++
+Only applies if you have the Remote Execution plug-in installed.
+endif::[]
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Last execution succeeded / No execution finished yet | OK | 0
+| Last execution failed | Error | 1
+| Unknown execution status | OK | 2 or 3
+| Last execution cancelled | OK | 4
+|===
 
-.Sub-statuses that map to the global *OK* status
+ifdef::satellite,orcharhino,katello[]
+Inventory::
+Indicates if the host is synchronized to {RHCloud}.
+{ProjectServer} performs the synchronization itself but only uploads basic information to {RHCloud}.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Host was not uploaded to your RH cloud inventory | Warning | 0
+| Successfully uploaded to your RH cloud inventory | OK | 1
+|===
 
-Active::
-During the last run, some resources were applied.
+Insights::
+Indicates if the host is synchronized to {RHCloud}.
+This synchronization is performend by the host.
+The host uploads more information than the {ProjectServer}.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Reporting | OK | 0
+| Not reporting | Error | 1
+|===
 
-Pending::
-During the last run, some resources would be applied but your configuration management integration was configured to run in `noop` mode.
+Errata::
```

Comment:
Errata should be "katello,satellite,orcharhino". please add "endif" here and another "ifdef:katello,satellite,orcharhino" below. see my other comment above.